LEGO1/mxdirectx/mxdirect3d.cpp

```diff
@@ -538,13 +547,15 @@ void MxDeviceEnumerate::BuildErrorString(const char* p_format, ...)
 	OutputDebugString(buf);
 }
 
+// FUNCTION: CONFIG 0x00401bf0
 // FUNCTION: LEGO1 0x1009c4f0
 HRESULT CALLBACK MxDeviceEnumerate::DisplayModesEnumerateCallback(LPDDSURFACEDESC p_ddsd, LPVOID p_context)
```

Comment:
Are these correct? Or do I need to put the following elsewhere?

```
// FUNCTION: CONFIG 0x00401bf0
MxDeviceEnumerate::DisplayModesEnumerateCallback
```

Reply:
That should be correct afaik. @disinvite can you confirm?

Reply:
The `CONFIG` annotations on the MxDirect3d functions all look fine. You don't need the leading zeroes on the hex address (to pad to 8) but I think the parser will allow it.

Reply:
My personal preference is to keep the leading zeros, but I'll change them if requested.